users.js

```diff
@@ -1696,5 +1692,5 @@ let Users = Object.assign(getUser, {
 	}, 1000 * 60 * 30),
 	socketConnect: socketConnect,
 });
-
+// @ts-ignore
```

Comment:
Why does this need to be ts-ignored?

Reply:
`[ts] Variable 'Users' is used before being assigned.` I started getting this error with the very first beta release of 3.0.0 I used,, and 3.0.1 still has it. I could never manage to fix it properly, so I had to just ignore it.